front/app/modules/commercial/representativeness/admin/components/Field/utils.ts

```diff
@@ -0,0 +1,199 @@
+// utils
+import { isNilOrError, NilOrError } from 'utils/helperUtils';
+
+// typings
+import { IUserCustomFieldOptionData } from 'modules/commercial/user_custom_fields/services/userCustomFieldOptions';
+import {
+  IReferenceDistributionData,
+  TDistribution,
+  TUploadDistribution,
+} from '../../services/referenceDistribution';
+
+// EXPORTS
+export type FormValues = Record<string, number | null>;
```

Comment:
This is where the form values are stored. If a field is enabled but the user hasn't typed in a number, the value is `null`. If the user has typed a number, the value is a `number`. If the field is disabled, its key is removed from the object (see `Field/index` component)